server/ui/src/store.js

```diff
@@ -75,12 +75,20 @@ const actions = {
 
   async login({ commit }, creds) {
     const data = await API.login(creds)
-    if (data.code === 401) {
+    if (data.code !== 200) {
       throw new Error(data.error.message)
     }
     return saveSession(commit, data)
   },
 
+  async loginSSO({ commit }, email) {
```

Comment:
What does `{ commit }` do/mean?

Reply:
Can it be removed?

Reply:
Yeah I can replace it with an unused var, but args in js are positional so that spot still needs to be filled. I'll just use `_`